runtime/src/main/jni/CallbackHandlers.cpp

```diff
@@ -317,8 +317,7 @@ void CallbackHandlers::CallJavaMethod(const Local<Object>& caller, const string&
         callerJavaObject = objectManager->GetJavaObjectByJsObject(caller);
         if (callerJavaObject.IsNull()) {
             stringstream ss;
-            ss << "No java object found on which to call \"" << methodName <<
-               "\" method. It is possible your Javascript object is not linked with the corresponding Java class. Try passing context(this) to the constructor function.";
+            ss << "Failed calling " << methodName << " on a " << className << " instance. The JavaScript instance no longer has available Java instance counterpart.";
```

Comment:
Shouldn't it be better if the error be handled during marshalling and end up in "Argument 'null' at index 0 can not be converted to Java instance" (or even better if it could be 'android.content.Context instance')? With source line and column pointing to the constructor call?